web/Gruntfile.js

```diff
@@ -9,9 +9,13 @@ module.exports = function (grunt) {
         require('load-grunt-tasks')(grunt);
     } else {
         // Use jit-grunt to only load necessary tasks for each invocation of grunt.
-        require('jit-grunt')(grunt);
+        require('jit-grunt')(grunt, {
+            ngtemplates: 'grunt-angular-templates'
+        });
     }
 
+    const ngTemplatesPaths = require('./node_modules/grunt-incuna-plugins/helper-functions/ng-templates-paths')();
```

Comment:
If you set a `main` file in that library's `package.json`, you can `require('grunt-incuna-plugins')` and use that to access `ng-templates-paths`. (One can recursively require all files in a folder and export them as their file names.)